### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by camerondgray**

### CODE REVIEW

**Comment by beastawakens** [comment]
Not strictly related to this readme but we should probably copy the plain `aws` and `gcp` README's to their R53/Google DNS equivalents, and amend if required.

**Review by ddollar — DISMISSED** [review]

File: aws-route53/main.tf [hunk 1]
```diff
@@ -23,6 +23,11 @@ variable "ssh_key" {
   default     = ""
 }
 
+variable "zone" {
```
Comment (ddollar) [inline aws-route53/main.tf]: is this rebased? this variable is already on master

Comment (camerondgray) [reply aws-route53/main.tf]: yes I reverted by previous change and rebased.

Comment (ddollar) [reply aws-route53/main.tf]: so why is this variable change still in the PR?

**Review by ddollar — APPROVED** [review]

**Pull request merged**